doc/admin/deployment.md

```diff
@@ -139,29 +128,22 @@ invocation of *daos\_server* and must be run as root. SCM modules will
 be configured into interleaved regions with memory mode set to
 "app-direct" with one set per socket (each module is assigned to socket
 and reports this via its NUMA rating). This configuration may require a
-reboot and *daos\_server* will exit on completion of the task.
+rebooy and *daos\_server* will exit on completion of the task.
 
 This can be done manually via the following commands:
 
 \# to verify there is non-volatile memory type
 
-\$ ipmctl show -a -topology | egrep 'Capac|MemoryType'
-
-  MemoryType=DCPM
-
-  Capacity=502.5 GiB
-
-  MemoryType=DCPM
+\# ipmctl show -a -topology | egrep 'Capac|MemoryType'  
+MemoryType=Logical Non-Volatile Device  
```

Comment:
where do you get this from?
shouldn't we be reporting DCPM devices?
[root@wolf-72 tanabarr]# ipmctl show -a -topology | egrep 'Capac|MemoryType'
   MemoryType=DCPM
   Capacity=502.5 GiB
   MemoryType=DCPM
   Capacity=502.5 GiB
   MemoryType=DCPM
   Capacity=502.5 GiB
   MemoryType=DCPM
   Capacity=502.5 GiB
   MemoryType=DCPM
   Capacity=502.5 GiB
   MemoryType=DCPM
   Capacity=502.5 GiB
   MemoryType=DCPM
   Capacity=502.5 GiB
   MemoryType=DCPM
   Capacity=502.5 GiB
   MemoryType=DDR4
   Capacity=8.0 GiB
   MemoryType=DDR4
   Capacity=8.0 GiB
   MemoryType=DDR4
   Capacity=8.0 GiB
   MemoryType=DDR4
   Capacity=8.0 GiB
   MemoryType=DDR4
   Capacity=8.0 GiB
   MemoryType=DDR4
   Capacity=8.0 GiB
   MemoryType=DDR4
   Capacity=8.0 GiB
   MemoryType=DDR4
   Capacity=8.0 GiB

Reply:
This is what I got from wolf-133:
root@wolf-133:~$ ipmctl show -a -topology | egrep 'Capac|MemoryType'
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=Logical Non-Volatile Device
   Capacity=502.6 GiB
   MemoryType=DDR4
   Capacity=16.0 GiB

Either MemoryType is changed from DCPM to this long name, or, this is a bug in DCPM driver?

Reply:
no sorry you are correct, wolf-72 which I was running on has ipmctl v1 whereas you were running v2